atomics/T1613/T1613.yaml

```diff
@@ -1,33 +1,66 @@
 ---
 attack_technique: T1613
-display_name: "Container and Resource Discovery"
+display_name: "Docker Container and Resource Discovery"
```

Comment:
The display name is correct here. Instead update it lower in the file :

atomic_tests:
- name:  <. >